app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughActivityPresenter.kt

```diff
@@ -20,24 +20,27 @@ class WalkthroughActivityPresenter @Inject constructor(
 ) : WalkthroughActivityListener {
   private lateinit var topicId: String
   private lateinit var binding: WalkthroughActivityBinding
+  private val walkthroughActivityViewModel by lazy {
```

Comment:
It should be `walkthroughViewModel`, as we have no other parts to differentiate from like fragment or any other part.

Reply:
Please see 
1 -> [comment1](https://github.com/oppia/oppia-android/pull/1911#discussion_r505202376) 
2 -> [comment2](https://github.com/oppia/oppia-android/pull/1911#discussion_r500759819)

we can't name it `walkthroughViewModel` and `viewModel`